cmd/controller/main.go

```diff
@@ -204,6 +206,10 @@ func main() {
 				},
 			},
 		},
+		Controller: config.Controller{
+			// ReconciliationTimeout is used as the timeout passed to the context of each Reconcile call.
+			ReconciliationTimeout: 30 * time.Second,
```

Comment:
Does that change mean that all our reconciler will timeout after 30 seconds? I think it's a good idea to have an explicit timeout, but maybe 30 seconds is too low as a default value

Reply:
bumped to 90 seconds as agreed